shell/platform/windows/accessibility_bridge_windows_unittests.cc

```diff
@@ -50,7 +50,7 @@ class AccessibilityBridgeWindowsSpy : public AccessibilityBridgeWindows {
 
   void SetFocus(std::shared_ptr<FlutterPlatformNodeDelegateWindows>
                     node_delegate) override {
-    focused_nodes_.push_back(node_delegate->GetAXNode()->id());
+    focused_nodes_.push_back(node_delegate);
```

Comment:
You can move the shared pointer to avoid unnecessary reference counting. It does the same thing but slightly faster :)

```suggestion
    focused_nodes_.push_back(std::move(node_delegate));
```